Sources/TrustSDK/ObjCWrapper/TrustSDKObjC.swift

```diff
@@ -0,0 +1,47 @@
+// Copyright © 2018 Trust.
+//
+// This file is part of TrustSDK. The full TrustSDK copyright notice, including
+// terms governing use, modification, and redistribution, is contained in the
+// file LICENSE at the root of the source code distribution tree.
+#if XAMARIN
```

Comment:
I don't think we need this `#if`

Reply:
without the #if the extension would be included in normal ios builds and the sign methods get overloaded. now only if i set the XAMARIN flag in the compiler to make a binding the extension gets added to the class

Reply:
Yes, I guess something needs to be different otherwise there are two definitions of the same thing. Let's leave it for now and when we add Result that will distinguish them and we can refactor.